src/ModuleManager.php

```diff
@@ -74,6 +73,7 @@ public function register(LibraryConfiguration $library)
 
             // Run the module's onRegister handler.
             $class::onRegister($this->g, $m);
+            $m->save($this->g->getEntityManager());
```

Comment:
No. It's just that after you register a module, you expect it to be persisted to the DB.